index.html

```diff
@@ -9,23 +9,24 @@
   <title>Portfolio setup and mobile</title>
 </head>
 <body>
-<section class="headline">

```

Comment:
- [ ] I kindly suggest that you make use of responsive widths (%) for your widths because currently there is a massive horizontal overflow that is hidden but this is causing the design to be way different from the Figma design.
- [ ] I also suggest that you make sure all your sections are aligned as seen from the Figma design.

- Kindly check the video below for more explanation. 

https://www.loom.com/share/3631969262b540048c3f5472068c400e

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="960" alt="image" src="https://user-images.githubusercontent.com/52934128/222201993-23d0ed49-568c-4edd-b3e2-3f1a33383f4d.png">  | <img width="589" alt="image" src="https://user-images.githubusercontent.com/52934128/222202641-1c31dab2-1a73-4464-aa32-ef21975809a1.png">  |